Apps/Playground/CMakeLists.txt

```
@@ -162,6 +180,31 @@ if(APPLE)
             set(CMAKE_Swift_COMPILER_FORCED TRUE)
             set(CMAKE_Swift_LANGUAGE_VERSION 4.0)
             enable_language(Swift)
+    elseif(VISIONOS)
+        set_target_properties(Playground PROPERTIES
+            MACOSX_BUNDLE true
+            MACOSX_BUNDLE_INFO_PLIST "${PLIST_FILE}"
+            XCODE_ATTRIBUTE_CLANG_ENABLE_OBJC_ARC YES
+            RESOURCE "${RESOURCE_FILES}"
+
+            XCODE_ATTRIBUTE_XROS_DEPLOYMENT_TARGET ${DEPLOYMENT_TARGET}
+            XCODE_ATTRIBUTE_PRODUCT_BUNDLE_IDENTIFIER "com.BabylonNative.Playground.visionOS"
+
+            XCODE_ATTRIBUTE_SWIFT_VERSION "5.0"
+            XCODE_ATTRIBUTE_SWIFT_OBJC_BRIDGING_HEADER "${CMAKE_CURRENT_LIST_DIR}/visionOS/LibNativeBridge.h"
+            XCODE_ATTRIBUTE_LD_RUNPATH_SEARCH_PATHS "@executable_path/Frameworks"
+            XCODE_ATTRIBUTE_FRAMEWORK_SEARCH_PATHS "$(inherited) $(SDKROOT)$(SYSTEM_LIBRARY_DIR)/Frameworks"
+            XCODE_ATTRIBUTE_ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES YES
+            XCODE_ATTRIBUTE_SWIFT_OBJC_INTEROP_MODE "objcxx"
+
+            # CMake seems to add a custom flag "-Wno-unknown-pragmas" to the Swift compiler. That flag is used for Clang,
+            # So we need to make sure we override it with nothing here in order to compile Swift.
+            XCODE_ATTRIBUTE_OTHER_SWIFT_FLAGS "")
+
+            # Swift support
+            set(CMAKE_Swift_COMPILER_FORCED TRUE)
+            set(CMAKE_Swift_LANGUAGE_VERSION 5.0)
+            enable_language(Swift)
```

These are indented too far, really through me off for a sec. I assume this is just a copy/paste of the same pre-existing formatting issue from the iOS section.
```suggestion
        set(CMAKE_Swift_COMPILER_FORCED TRUE)
        set(CMAKE_Swift_LANGUAGE_VERSION 5.0)
        enable_language(Swift)
```

---

Fixed for both sections